docs/migration/migrate_7_0.asciidoc

```diff
@@ -97,13 +97,6 @@ considered unique based on its persistent UUID, which is written to the path.dat
 
 *Impact:* The logging.useUTC flag will have to be replaced with a timezone id.  If set to true the id is `UTC`.
 
-[float]
```

Comment:
__note:__ I thought it'd make sense to join these two notes into one, let me know if you have a better wording.